quickstarts/aws/amazon-chatbot/config.yml

```diff
@@ -0,0 +1,31 @@
+slug: aws-chatbot
+description: |-
+  ## What is AWS Chatbot?
+  AWS Chatbot is an AWS service that enables DevOps and software development teams to use messaging program chat rooms to monitor and respond to operational events in their AWS Cloud. AWS Chatbot processes AWS service notifications from Amazon Simple Notification Service (Amazon SNS), and forwards them to chat rooms so teams can analyze and act on them immediately, regardless of location.
+  ### Get started!
+  Start monitoring AWS Chatbot by using AWS Kenesis Firehose to send logs to New Relic! Take a look at our documentation for information on how to set this integration.
+  ### More info
+  Check out the [documentation](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-chatbot-monitoring-integration/) to learn more about New Relic monitoring for AWS Chatbot.
+summary: |-
+  Monitor AWS Chatbot by streaming Firehose logs into New Relic
+icon: logo.svg
+level: New Relic
+authors:
+  - New Relic
+title: AWS Chatbot
+documentation:
+  - name: AWS Chatbot installation docs
+    description: |
+      Monitor AWS Chatbot by connecting AWS to New Relic.
+    url: >-
+      https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-chatbot-monitoring-integration/
+keywords:
+  - aws
+  - amazon web services
+  - networking
+dashboards:
+  - AWS-chatbot
+installPlans:
```

Comment:
Since the installPlan was removed, we'll also need to remove this reference.

Reply:
@MichelLosier , addressed, i removed installPlans.